### PR TITLE
Add mongo-express to review mongodb.

### PR DESCRIPTION
**Pull request by qzs21**
This <https://github.com/andzdroid/mongo-express> in docker: <https://github.com/knickers/mongo-express>

### CODE REVIEW

File: formulas/mongo-express [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+run --detach \
+    --publish $me_dock_port:$me_dock_port \
+    --link dock-mongodb:mongo \
+    --name $me_dock_name \
+    knickers/mongo-express
+
+# Open in browser (OS X)
+container_ip=$(get_ip)
+open http://$container_ip:$me_dock_port
+echo "Now, mongo-express is show in browser."
```
Comment (bripkens) [inline formulas/mongo-express]: Please remove the last three lines. It is not dock's responsibility to open URLs in the browser.

File: formulas/mongo-express [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source "${BASH_SOURCE%/*}/../common"
+
+# Configs
+me_dock_name=dock-mongo-express
```
Comment (bripkens) [inline formulas/mongo-express]: Can you rename this to `container_name`?

File: formulas/mongo-express [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+me_dock_port=8081
```
Comment (bripkens) [inline formulas/mongo-express]: Same here: `container_port`.

File: formulas/mongo-express [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+# Stop Container
+force_stop $me_dock_name
+
+# run Container
+run --detach \
+    --publish $me_dock_port:$me_dock_port \
+    --link dock-mongodb:mongo \
+    --name $me_dock_name \
+    knickers/mongo-express
```
Comment (bripkens) [inline formulas/mongo-express]: Can we reference a stable docker image tag? We have seen multiple cases where images were updated and dock formulas broken in the process. A tag would help avoiding this.

File: formulas/mongo-express (hunk 3 above)
Comment (iineva) [inline formulas/mongo-express]: OK, wait a few minutes.

**Comment by bripkens** [comment]
Thank you very much for this formula @qzs21! I have a few minor remarks. Once those are resolved, I'll gladly accept this.

**Comment by bripkens** [comment]
Looks good, thanks @qzs21!

**Comment by iineva** [comment]
i finished my coding few hours ago, but my networking is really worse, i had finished pushing the docker image just now!

**Pull request merged**